### ISSUE TITLE
Offline option fails dialyzer check

### ISSUE THREAD

**Issue by lornatumuhairwe** [launchdarkly/erlang-server-sdk#22]
**Describe the bug**
An error is returned when dialyzer runs on an application starting ldclient in offline mode.

**To reproduce**
Invoke `ldclient.start_instance/2` with `:offline => true` in an application using dialyzer.

**Expected behavior**
Invoking  `ldclient.start_instance/2` with `:offline => true` shouldn't return any type errors. 

**Logs**
```
Total errors: 1, Skipped: 0, Unnecessary Skips: 0
done in 0m14.53s
lib/feature_flag/application.ex:15:call
The function call will not succeed.

:ldclient.start_instance(string(), %{:offline => true})

will never return since it differs in arguments with
positions 2nd from the success typing arguments:

(
  string(),
  atom()
  | %{
      :base_uri => string(),
      :events_capacity => pos_integer(),
      :events_dispatcher => atom(),
      :events_flush_interval => pos_integer(),
      :inline_users_in_events => boolean(),
      :polling_interval => pos_integer(),
      :polling_update_requestor => atom(),
      :private_attributes => :all | [atom() | binary()],
      :storage_backend => atom(),
      :stream => boolean(),
      :stream_uri => string(),
      :user_keys_capacity => pos_integer()
    }
)

________________________________________________________________________________
done (warnings were emitted)
Halting VM with exit status 2
```

**SDK version**
1.0.0-beta2

**Language version, developer tools**
Elixir 1.9.4

**OS/platform**
Docker container running alpine-3.11 linux inside Docker for Mac 2.2.0.5(43884)

**Additional context**
This seems to be because offline key is not defined in the type options [here](https://github.com/launchdarkly/erlang-server-sdk/blob/master/src/ldclient_instance.erl#L17)

**Comment by zurab-darkly** [comment]
@lornatumuhairwe thank you for the detailed report. This will be fixed in the next release.

**Comment by zurab-darkly** [comment]
This is now fixed in 1.0.0-beta3 release, thank you again for reporting.
